simpleflow/swf/executor.py

```diff
@@ -413,6 +421,35 @@ def get_future_from_external_workflow_event(self, a_task, event):
 
         return future
 
+    def _get_future_from_timer_event(self, a_task, event):
+        """
+        Maps a timer event to a Future with the corresponding state.
+
+        :param a_task: Timer task; unused.
+        :type a_task: TimerTask
+        :param event: Timer event
+        :type event: dict[str, Any]
+        :return:
+        :rtype: futures.Future
+        """
+        future = futures.Future()
+        if not event:
+            return future
+        state = event['state']
+        if state == 'started':
+            future.set_running()
+        elif state == 'fired':
+            future.set_finished(None)  # FIXME send control?
```

Comment:
I was just wondering if there's something in the message we could send; but `control` seems wrong. Will just remove the comment 🙂